content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -410,7 +410,7 @@ To disallow access, the service would return:
   "kind": "ImageReview",
   "status": {
     "allowed": false,
-    "reason": "image currently blacklisted"
+    "reason": "image currently blocklisted"
```

Comment:
Watch out for changing documentation without also changing the code that generates data. Revising the docs here doesn't fix the code.

Reply:
I think this one is safe to change since it's a suggestion for what the external imagereview implementation should return.

Reply:
Ah, good point.
How about a reword: `"reason": "image blocked by malware scan"`

IMO _blocklisted_ is a bit of a bowdlerization: it weakens the text when a different wording could work better.